src/components/FavoriteItem/FavoriteItem.js

```diff
@@ -0,0 +1,148 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+
+import classNames from 'classnames'
+import { ButtonWithConfirm } from 'components'
+import { JText, JCard, JIcon, JTooltip } from 'components/base'
+
+type Props = {
+  editItem: () => void,
+  deleteItem: () => void,
+  address: Address,
```

Comment:
better to change `editItem` -> `edit` & `deleteItem` -> `remove` (delete is reserved word)
to avoid redundant word 'Item'